src/infer_model.py

```diff
@@ -0,0 +1,115 @@
+import os
+import pandas as pd
+import json
+from transformers import AutoTokenizer, AutoModelForSeq2SeqLM, pipeline
+import torch
+from vllm import LLM, SamplingParams
+from dotenv import load_dotenv
+from load_dataset import get_swear_words, get_prompts, get_model_inferences
+from calculate_metrics import evaluation_script_case_1, evaluation_script_case_2, calculate_percentage_case_1, calculate_percentage_case_2
+
+load_dotenv()
```

Comment:
I think the .dot env file is not pushed right now

Reply:
But it has the API key, so it shouldn't be pushed.